.phraseapp.yml

```diff
@@ -0,0 +1,18 @@
+phraseapp:
```

Comment:
I think we don't need to use Phraseapp for IC. This file can be removed

Reply:
@hanam1ni Fixed in [`ece0c06` (#26)](https://github.com/suho/google-search-ruby/pull/26/commits/ece0c064d824aa98eb9589fd826787aadba2bd60)